# if you change this yaml file run the load_compensation script in the script directory with normal python
Motors:
    left_motor:
        x: 60
        y: -25
        z: 10
    right_motor:
        x: 60
        y: 25
        z: 10
ForcePoints:
    fp_lm:
        x: 65
        y: 12
        z: 0
    fp_rm:
        x: 65
        y: -12
        z: 0
max_steering_angle: 30
